.travis.yml

```diff
@@ -1,39 +1,29 @@
-notifications: # set notification options
-  email:
-    recipients:
-      - mgopez95@gmail.com
-      - peter.kruczkiewicz@gmail.com
-
-    # change is when the repo status goes from pass to fail or vice versa
-    on_success: change
-    on_failure: change
-
 language: python
 python:
-  - '3.6'
+- '3.6'
 os:
-  - linux
-#  - osx
-
-# Whitelist of branches to run CI Testing on.
+- linux
 branches:
   only:
-    - master
-    - development
-
-# Blacklist of branches to not run CI testing on.
-# branches:
-#  except:
-#   - dontrunme 
-#   - test 
-
-before_install:
-  - sudo apt-get -qq update;
-
-install:
-  - python setup.py install
-  #Install pytest for testing.
-  - pip install pytest
-
-script:
-  - pytest
+  - master
+  - development
+install: pip install -U tox-travis
+script: tox
+deploy:
+  provider: pypi
+  distributions: sdist bdist_wheel
+  user: peterk87
+  deployment: bio-hansel
+  password:
+    secure: feBsWxq8pSl/Eisu262EJH3E3JYgeWVmmwHCHut1474BxqXDvoUOGQmD7WoxwLCUVwvTE9gq37KZhYA4QDBwGI2vTwEegEeZGEo6ivPPO68TEkEFh00HwvXmFnYjllTtVISkUpBiUT8bYXdcPzR66bTfig+tZvlKIMATbHhzwFrWh1dkWkyiUojUaD4T861M0n774B0T8nyS4ppb7GT4CfkJXm9PpGMNWu04Cv0U2tTAIABT0OvvrstP1o1XFvOLNP3KI7aV/Onf24r3D5/jqtQN5ET90HFqvpoumRQi4XnJ10RWLySSOIlJ8HEE7giHvFz//TAGA4e3rBB4B0POun+qtAlriH2aq62vpBhm4jLZQOI8ARm6G1GPBKXZq+LnFJmQKFritYpNUubdpsaCK1ykoDgVOQh9FrAVXTcWvN6PPPwbrwZ2L5DkKsKyYK9QdtgM7Yhhr3cT2tM74jPQAeh9ky7iow4ltTlX/4NNegXwVxn7Gmn8Dkeg5dFeh/CcfaaHKSd3Se2DUe3vPldoQgkNJSUyFL65MvfftQP/pRRpLfPqkrHl6QjhCTdLLUvTTtyQtkvO5a3vNKtbScMK+epvGKrwSSPBIckYS0BUWHM/KoswnR+VPWh5wQmJWLHliukuseCx7qXJkABuZNsl8YtFXB+uCXGV9eoErR4sD5c=
+  on:
+    tags: true
+    repo: phac-nml/bio_hansel
+    python: 3.6
+notifications:
+  email:
+    recipients:
+    - mgopez95@gmail.com
+    - peter.kruczkiewicz@gmail.com
```

Comment:
Perhaps add myself and Darian on the notification list.

Reply:
Added you and Darian's emails in caa77d81bc03f2d769714d7163a192ee7b2e09d0